charts/aspnetcore/templates/deployment.yaml

```diff
@@ -35,9 +35,6 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ include "aspnetcore.serviceAccountName" . }}
-      {{- if .Values.volumes }}
```

Comment:
This overrided the entire `volumes` key making it super un-flexible and even clashed with `extraVolumes` if both were set. It should never have been included.